doc/sphinx/source/animations.rst

```diff
@@ -90,6 +100,71 @@ We also support mipmaps here -- name the files ``walk_ne_0.5_00.png``,
 ``walk_ne_1_01.png`` etc. for scale `1` and so on.
 
 
+.. _animations_spritesheets:
+
+Spritesheet Animations
+----------------------
+
+The same map object can have a mix of file and spritesheet animations.
+For converting a file animation to a spritesheet animation, use the command line
+to call ``./wl_create_spritesheet`` from the Widelands program directory.
+This will print the command line options to use for converting your animation.
+
+After conversion, you will need to delete the old files and copy over the new files,
+then delete the old animation code in the map object's ``init.lua`` file and add the new code.
+The tool will print the new code to the console for you, so all you need to do is copy/paste
+and then add any missing optional parameters back in.
+
+Note that except for flag and frontier animations, the table for spritesheets is
+called ``spritesheets``, not ``animations``!
```

Comment:
It is confusing that spritesheets are sometimes in a table that looks like a file anim table. Please rename the tribe´s "animations" table to "spritesheets"

Reply:
Ah, so this is mixed up in TribeDescr loading code. IMHO it would be cleaner if we have one table "spritesheets" for the spritesheets and one table "animations" for nonpacked anims (both optional) in the TribeDescr, like for the MapObjects.

Reply:
Done.